tests/test_validity_checker.py

```diff
@@ -103,62 +107,124 @@ def test_upload_file(client):
     assert error_description == "No filesize sent"
 
 
-def test_test_names(client):
-    for geode_object in geode_objects_list.keys():
-        print(f"{geode_object=}")
-        inputs = geode_objects_list[geode_object]["input"]
-
-        for input in inputs:
-            for input_extension in input.list_creators():
-                print(f"{input_extension=}")
-                filename = f"test.{input_extension}"
-                file = base64.b64encode(
-                    open(f"./tests/data/test.{input_extension}", "rb").read()
-                )
-                filesize = int(os.path.getsize(f"./tests/data/test.{input_extension}"))
-
-    for geode_object in geode_objects_list.keys():
-        # Normal test with all objects
-        response = client.post(
-            f"{base_route}/tests_names", data={"geode_object": geode_object}
-        )
-        assert response.status_code == 200
-        model_checks = response.json["model_checks"]
-
-        assert type(model_checks) is list
-        for model_check in model_checks:
-            assert type(model_check) is dict
-            is_leaf = model_check["is_leaf"]
-            route = model_check["route"]
-            children = model_check["children"]
-            assert type(is_leaf) is bool
-            assert type(route) is str
-            assert type(children) is list
-            for check in children:
-                assert type(check) is dict
-                if check["is_leaf"] == True:
-                    print("is_leaf")
-                    response_test = client.post(
-                        f"{base_route}/inspect_file",
-                        data={
-                            "object": "BRep",
-                            "filename": "corbi.og_brep",
-                            "test": check["route"],
-                        },
-                    )
-
-                    assert response_test.status_code == 200
-                else:
-                    print("not is_leaf")
-
-
-def test_inspect_file(client):
-    # Test with file
-    response = client.post(
-        f"{base_route}/inspect_file",
-        data={
-            "file": base64.b64encode(open("./tests/data/test.og_brep", "rb").read()),
-            "filename": "corbi.og_brep",
-            "filesize": os.path.getsize("./tests/data/test.og_brep"),
-        },
-    )
+# def upload_files(client):
```

Comment:
to remove/clean?

Reply:
This tests are failing in github but they work in local, it's because they don't find the test files located in the /upload directory